frontend/src/App.css

```diff
@@ -6,7 +6,8 @@ body {
 }
 .App {
   text-align: center;
-  background: linear-gradient(-45deg,#EE7752,#E73C7E,#23A6D5,#23D5AB);
+   /* background: linear-gradient(-45deg,#EE7752,#E73C7E,#23A6D5,#23D5AB); */
+  background: linear-gradient(-45deg,black,#4f4f4f,black,#4f4f4f); 
```

Comment:
You can keep the colors you will always use in a variable and use them everywhere. It also makes it easier to change color.